include/timer/timer_config.h

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
we have the timer config in include/timer/timer_config.h, but the main config is still in include/logger. maybe we should have a top level config directory to organize the location of configs?